site/content/docs/user/nvidia-dranet.md

```diff
@@ -0,0 +1,460 @@
+    inet 10.68.3.41/24 brd 10.68.3.255 scope global eth0
+       valid_lft forever preferred_lft forever
+4: gpu0rdma0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 8896 qdisc mq state UP group default qlen 1000
+    link/ether 06:c4:a0:25:7e:01 brd ff:ff:ff:ff:ff:ff
+    inet 192.168.1.5/32 scope global gpu0rdma0
+       valid_lft forever preferred_lft forever
+5: gpu1rdma0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 8896 qdisc mq state UP group default qlen 1000
+    link/ether e2:f1:94:78:7e:04 brd ff:ff:ff:ff:ff:ff
+    inet 192.168.2.5/32 scope global gpu1rdma0
+       valid_lft forever preferred_lft forever
+```
+
+And now we run NCCL again.
+
+```sh
+root@nccl-gib-test-0:/diagnostic# /usr/local/gib/scripts/run_nccl_tests.sh   -t all_gather -b 1K -e 8G   nccl-gib-test-0 10.68.5.40 -g 2
+Initializing SSH...
+Hello from nccl-gib-test-0
+Warning: Permanently added '[10.68.5.40]:222' (ECDSA) to the list of known hosts.
+Hello from 10.68.5.40
+Generating hostfiles for 2 hosts:
+nccl-gib-test-0
+10.68.5.40
+# nThread 1 nGpus 1 minBytes 1024 maxBytes 8589934592 step: 2(factor) warmup iters: 50 iters: 100 agg iters: 1 validation: 1 graph: 0
+#
+# Using devices
+#  Rank  0 Group  0 Pid   3521 on nccl-gib-test-0 device  0 [0000:8f:00] NVIDIA B200
+#  Rank  1 Group  0 Pid   3514 on nccl-gib-test-0 device  1 [0000:90:00] NVIDIA B200
+#  Rank  2 Group  0 Pid   2071 on nccl-gib-test-1 device  0 [0000:8f:00] NVIDIA B200
+#  Rank  3 Group  0 Pid   2076 on nccl-gib-test-1 device  1 [0000:90:00] NVIDIA B200
+NCCL version 2.26.6+cuda12.8
+#
+#                                                              out-of-place                       in-place
+#       size         count      type   redop    root     time   algbw   busbw #wrong     time   algbw   busbw #wrong
+#        (B)    (elements)                               (us)  (GB/s)  (GB/s)            (us)  (GB/s)  (GB/s)
+...
+   268435456      16777216     float    none      -1   3059.9   87.73   65.80      0   3134.1   85.65   64.24      0
+   536870912      33554432     float    none      -1   5886.9   91.20   68.40      0   6082.2   88.27   66.20      0
+  1073741824      67108864     float    none      -1    12452   86.23   64.67      0    11550   92.97   69.73      0
+  2147483648     134217728     float    none      -1    24426   87.92   65.94      0    23694   90.63   67.97      0
+  4294967296     268435456     float    none      -1    47420   90.57   67.93      0    47000   91.38   68.54      0
+
+  8589934592     536870912     float    none      -1    96100   89.39   67.04      0    94047   91.34   68.50      0
+```
+
+They now connect!
+
+#### Conclusion
+
+Using both DRANet and the Nvidia DRA libraries in combination is a way to quickly allocate both GPUs and RDMA devices in order to create interconnected workloads that can span multiple nodes. This can be used the create workloads that span multiple nodes and take advantage of spare resources on nodes. 
```

Comment:
you should add another case where the GPU and nic are not aligned and show the results, there should be an important impact in performance

Reply:
Let me do that, need to spin up another cluster.